packages/app-project/components/Head/HeadContainer.js

```diff
@@ -0,0 +1,72 @@
+import { inject, observer } from 'mobx-react'
+import { withRouter } from 'next/router'
+import { shape, string } from 'prop-types'
+import React, { Component } from 'react'
+import urlParse from 'url-parse'
+
+import Head from './Head'
+
+function storeMapper (stores) {
+  const { project } = stores.store
+  return {
+    project
+  }
+}
+
+@withRouter
+@inject(storeMapper)
+@observer
+export default class HeadContainer extends Component {
+  getProjectDescription () {
+    return this.props.project.description || undefined
+  }
+
+  getProjectBackgroundImage () {
+    return this.props.project.background.src || undefined
+  }
+
+  getProjectTitle () {
+    const { display_name, slug } = this.props.project
+    return display_name || slug || undefined
+  }
+
+  getProjectTwitterUsername () {
+    const { urls } = this.props.project
+    const twitter = urls.find(url => url.site &&
+      url.site.includes('twitter.com'))
+    if (twitter && twitter.path) {
+      return (twitter.path.startsWith('@'))
+        ? twitter.path
+        : `@${twitter.path}`
+    } else {
+      return undefined
+    }
+  }
+
+  getProjectUrl () {
+    // TODO: Set this from an environment variable
+    const host = 'http://localhost:3000'
+    const { slug } = this.props.project
+    return `${host}/projects/${slug}`
+  }
+
+  render () {
+    return (
+      <Head
+        description={this.getProjectDescription()}
+        ogImage={this.getProjectBackgroundImage()}
```

Comment:
Should this be the project avatar?

Reply:
I thought the background is likely to be a large image, and lend itself to `summary-card-with-large-image`: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image

Reply:
#425 says use the avatar, so it's worth checking this with project owners.

Reply:
Avatar it is then, with background as a fallback

Reply:
👍 In theory, you can't launch a project without having both, so all the live projects should have them.

Reply:
Cool, this should be ready to go now